en/docs/setup/configuring-a-read-write-active-directory-user-store.md

```diff
@@ -32,8 +32,11 @@ should be active_directory_unique_id.
 <tr class="odd">
 <td>base_dn</td>
 <td>User Search Base</td>
-<td>DN of the context or object under which the user entries are stored in the user store. When the user store searches for users, it will start from this location of the directory<br />
-Sample values: ou=Users,dc=wso2,dc=org</td>
+<td>DN of the base directory where Users container with user entries is stored in the user store.
+When the userstore searches for users, it will start from this location of the directory and search inside the Users container for users.<br />
```

Comment:
```suggestion
When the user store searches for users, it will start from this location of the directory and search inside the <b>Users</b> container for users.<br />
```